webpack/utils/buildThemeJSON.babel.js

```diff
@@ -0,0 +1,2 @@
+require('babel-register'); // eslint-disable-line import/no-extraneous-dependencies
+module.exports = require('./buildThemeJSON');
```

Comment:
we should remove the lockfile dance if we are only building it once

Reply:
I can do that.